INSS.EIIR.DailyExtract/FnHealthCheckPing.cs

```diff
@@ -10,12 +11,18 @@ namespace INSS.EIIR.DailyExtract
 {
     public class FnHealthCheckPing
     {
+
+        private readonly ILogger<FnHealthCheckPing> _logger;
+
+        public FnHealthCheckPing(ILogger<FnHealthCheckPing> logger)
+        {
```

Comment:
do you need logger stuff ? its not used

Reply:
No it is not needed anymore, though have left it in as an example